products/bigquery/terraform.yaml

```diff
@@ -88,6 +88,67 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       datasetId: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+  Job: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/jobs/{{job_id}}"]
+    skip_delete: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "bigquery_job_query"
+        primary_resource_id: "job"
+        vars:
+          job_id: "job_query"
+          account_name: "bqowner"
+        ignore_read_extra:
+          - "etag"
```

Comment:
Is the etag returned different between create and import?

Reply:
for one of the tests only, the `extract` test.